src/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj

```diff
@@ -23,6 +23,9 @@
   <PropertyGroup Condition="'$(TargetFramework)' == '$(TargetFrameworkNetStandard)'">
     <DefineConstants>$(DefineConstants);NETSTANDARD_RUNTIME</DefineConstants>
   </PropertyGroup>
+  <PropertyGroup>
+    <SignAssembly>true</SignAssembly>
```

Comment:
This is not needed, we add it to all projects via Directory.Build.Props

Reply:
pls. remove

---
In reply to: [262455207](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/939#discussion_r262455207) [](ancestors = 262455207)